### PR TITLE
 No duplicate heading

### PR DESCRIPTION
**Pull request by Nazar-Ant**
Changes to the English version of `hosting/heroku.md` and `plugins/keyboard.md` should be translated after approval.

# Specific problems:
## Spanish:
On line 202 of `es/guide/filter-queries.md` you have a duplicate header because the header seems to be translated incorrectly on line 123.
## Indonesian:
On line 1116 of `id/plugins/conversation.md` you have a duplicate header because the header seems to be translated incorrectly on line 1031.

### CODE REVIEW

File: site/docs/hosting/heroku.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ https://api.telegram.org/botabcd:1234/setWebhook?url=https%3A%2F%2Fgrammybot.her
 Use [Webhook Reply](../guide/deployment-types#webhook-reply) for more efficiency.
 :::
 
-### Creating `bot.ts`
+### Creating `bot.ts` for a Webhooks
```
Comment (roj1512) [inline site/docs/hosting/heroku.md:168]:
```suggestion
### Creating `bot.ts` (Webhooks)
```

File: site/docs/hosting/heroku.md [hunk 2]
```diff
@@ -231,7 +231,7 @@ If you want to do this once an hour, you can do that easily.
 That's something you cannot control with webhooks.
 If your bot gets flooded with messages, you will see a lot of webhooks requests, however, you can more easily limit the rate of updates to process with long polling.
 
-### Creating `bot.ts`
+### Creating `bot.ts` for a Long Polling
```
Comment (roj1512) [inline site/docs/hosting/heroku.md:234]:
```suggestion
### Creating `bot.ts` (Long Polling)
```

File: site/docs/plugins/keyboard.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ Naturally, all other methods that send messages other than text messages support
 For example, you can edit a keyboard by calling `editMessageReplyMarkup`, and passing the new `InlineKeyboard` instance as `reply_markup`.
 Specify an empty inline keyboard to remove all buttons underneath a message.
 
-### Responding to Clicks
+### Responding to Clicks on the Inline Keyboard
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:97]:
```suggestion
### Responding to Inline Keyboard Clicks
```

File: site/docs/plugins/keyboard.md [hunk 4]
```diff
@@ -282,7 +282,7 @@ new Keyboard()
 It does not matter whether you call `selected` first, last or somewhere in between.
 The result will always be the same.
 
-### Responding to Clicks
+### Responding to Clicks on the Custom Keyboard
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:285]:
```suggestion
### Responding to Custom Keyboard Clicks
```

**Review by roj1512 — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Comment by roj1512** [comment]
Is Chinese fine?

**Comment by Nazar-Ant** [comment]
Yes

**Comment by Nazar-Ant** [comment]
Hold on with that. There is still something that needs to be translated into Chinese. I meant that there are no specific problems for Chinese.

**Comment by roj1512** [comment]
Oh alright

**Comment by CikiMomogi** [comment]
Unfortunately, my PC is currently broken. I won't be able to access it in the near future.

**Comment by roj1512** [comment]
No worries, will do it myself when this is done!

**Comment by Nazar-Ant** [comment]
@habemuscode you have not fixed your specific problem (see issue description).

**Comment by habemuscode** [comment]
> @habemuscode you have not fixed your specific problem (see PR description).

[3778d9f](https://github.com/grammyjs/website/pull/908/commits/3778d9f287891edf955ef17123e602852f35a4b1)

I found that part totally out of sync, pushed fix. Thanks

**Review by roj1512 — APPROVED** [review]
Thanks @habemuscode!

**Pull request merged**